src/main/scala/com/github/unchama/seichiassist/listener/VotingFairyListener.scala

```diff
@@ -156,8 +161,12 @@ object VotingFairyListener {
       }
 
       // りんご所持数で値変更
-      val gachaimoObject = MineStackObjectList.findByName("gachaimo").unsafeRunSync().get
-      val l = playerdata.minestack.getStackedAmountOf(gachaimoObject)
+      val gachaimoObject =
```

Comment:
現在のガチャりんごが歴史的経緯によりDB上で`gachaimo`という文字列において特定されるだけなので、`gachaimo`にこだわる必要はありません
```suggestion
      val gachaRingo =
```